README.md

```diff
@@ -1,2 +1,22 @@
 # password-generator
 Generate passwords given criteria
+
+<h1>Description:-</h1>
+Created functions in javascript which allow for the generation of a password given user defined criteria.
+
```

Comment:
As above: More description about this project would be nice.

Reply:
👍